package.json

```diff
@@ -18,16 +18,18 @@
     ]
   },
   "scripts": {
-    "build-admin-panel": "yarn workspace @tupaia/admin-panel build",
-    "build-internal-dependencies": "./scripts/bash/buildInternalDependencies.sh",
-    "build-web-frontend": "yarn workspace @tupaia/web-frontend build",
+    "build": "build:internal-dependencies && build:non-internal-depenedencies",
+    "build:internal-dependencies": "./scripts/bash/buildInternalDependencies.sh",
+    "build:non-internal-dependencies": "./scripts/bash/buildNonInternalDependencies.sh",
+    "build:admin-panel": "yarn workspace @tupaia/admin-panel build",
+    "build:web-frontend": "yarn workspace @tupaia/web-frontend build",
     "download-env-vars": "./scripts/bash/downloadEnvironmentVariables.sh",
     "dump-database": "yarn workspace @tupaia/database dump-database --target ../../",
     "migrate": "yarn workspace @tupaia/database migrate",
     "migrate-create": "yarn workspace @tupaia/database migrate-create",
     "migrate-down": "yarn workspace @tupaia/database migrate-down",
     "package-json-lint": "npmPkgJsonLint --configFile .npmpackagejsonlintrc.json .",
-    "postinstall": "test -n \"$SKIP_BUILD_INTERNAL_DEPENDENCIES\" || yarn build-internal-dependencies",
+    "postinstall": "build",
```

Comment:
Interestingly for me this command fails with `/bin/sh: 1: build: not found`

I had to add `yarn` at the start here and for the `build:internal-dependencies && build:non-internal-depenedencies` line

Reply:
Actually there's a number of places where the `yarn` is now omitted and therefor the command fails. Not sure why this is happening on mine and not yours?

Reply:
Updated